Limelight/Input/StreamView.m

```diff
@@ -356,7 +356,7 @@ - (void)touchesBegan:(NSSet *)touches withEvent:(UIEvent *)event {
                 keyInputField.text = @"0";
 #if !TARGET_OS_TV
                 // Prepare the toolbar above the keyboard for more options
-                UIToolbar *customToolbarView = [[UIToolbar alloc] initWithFrame:CGRectMake(0, 0, self.bounds.size.width, 44)];
+                UIToolbar *customToolbarView = [[UIToolbar alloc] initWithFrame:CGRectMake(0, 0, self.bounds.size.width > 1500 ? self.bounds.size.width : 1500, 44)];
```

Comment:
How did you come up with 1500 here?

Reply:
By test :(
Didn't find a solution to calculate the width before drawing. I am not familiar with programing on iOS. Any suggestion?

Reply:
> How did you come up with 1500 here?

New pull request has removed this magic number by correctly calculating the toolbar width. For completeness, it also adds F7-F12 to the toolbar.
The new UI layout also makes the "cross" button on another level to make it always visible.